### ISSUE TITLE
Pre- and post-ringCT?

### ISSUE THREAD

**Issue by b-g-goodell** [noncesense-research-lab/tx_in_out_distribution#2]
Would be nice to have an up-to-date data set that excludes pre-RingCT outputs.

**Comment by neptuneresearch** [comment]
Ok I can split it.

edit for self reference: 
moneromooo> To get rct only, you get txes with version 2.
1220516 is when rct txes started being allowed.

**Comment by b-g-goodell** [comment]
Thanks! Ring CT was implemented in block #1220516 in January 2017. After September 2017, this feature became mandatory for all transactions on the network.  https://ww.getmonero.org/resources/moneropedia/ringCT.html

**Comment by neptuneresearch** [comment]
Thanks for your patience, I've been trying to get back to this.

So, to be specific, the transaction versions are

```
1 = signatures (pre-RingCT)
2 = rct_signatures (RingCT)
```

and regarding heights and transaction versions, I found these categories of transactions:

```
(A) Before height 1220516, transaction version 1
(B) After height 1220516, transaction version 1
(C) After height 1220516, transaction version 2
```

Would basing the distributions on transaction list (C) satisfy what you need with "exclude pre-ringCT outputs", or are you implying any more criteria than just "transaction version 2"? -- I wasn't sure if you meant anything more complicated by that word "outputs" than "the transaction version is 2".

**Comment by b-g-goodell** [comment]
You got it perfectly correct.

**Comment by neptuneresearch** [comment]
Ok.  Thanks again for patience! 

I added a distribution for RingCT only,  and threw in Non-RingCT anyway.

- ```ringct``` is (C) above
- ```nonringct``` is (A) and (B)

I skipped the master transaction list CSV this time because it's a big file (everything else is tiny) and started to feel unnecessary; I can add it if you do want.

That should be it for this issue. Enjoy!  or re-open if I missed something.
